src/components/Home/ActivityTicker.tsx

```diff
@@ -58,6 +88,7 @@ export default function ActivityTicker() {
         <Heading className="ActivityTicker__Title" size="3xs" weight="normal">
           {t('page.home.activity_ticker.title')}
         </Heading>
+        <ActivityTickerFilter setFilterState={setFilterState} filterState={filterState} refetch={refetch} />
```

Comment:
I don't like the pattern of passing a set state as a prop.
If this is where the state is used, why don't you handle the setFilterState logic from this side?

Something like...
```
const handleEventTypeClick = (event: EventType) => {
  setFilterState(prev => ({ ...prev, [event]: !prev[event] })
}
```

Reply:
I went for a different approach bc I needed two states, the ticked events in the filter box, and the actual filters. 
This is for being able to click outside the box or closing the filters without applying the changes to the search, unless you click apply.